core/model/modx/mysql/modelement.map.inc.php

```diff
@@ -38,6 +74,25 @@
       'default' => 0,
     ),
   ),
+  'indexes' =>
```

Comment:
I think it makes sense to add indexes for fields `createdby` and `editedby` for further use in widgets and extras when elements should be selected by author or editor.

Reply:
No need index all fields but for these fields this is obvious. As it works for resources already. 
Implementing it later will require additional work for reviewing, merging and releasing a new version that takes time and resources.

Reply:
Okay, no problem.